### ISSUE TITLE
Qwik double build generates empty css file when mangling is enabled

### ISSUE THREAD

**Issue by Machy8** [stylify/packages#266]
### Describe the bug

.

### Reproduction

.

### System Info

```shell
.
```

**Comment by petersrule** [comment]
Yes this is a big problem. It used to just generate an empty CSS file (global.css). Then I turned off the mangling(mangleSelectors: false) in the compiler and then it worked for a while, and then it started just messing up the CSS file referenced above so it didn't work anymore.  now it the Qwik dev server won't get started fully to serve files. I think I will have to go back to Tailwind.

**Comment by Machy8** [comment]
Hi @petersrule, do you have any other configuration? Like matching/ignore areas? I will look into it. I though, this is caused randomly, but it seems it's not.

**Comment by petersrule** [comment]
I just have the standard Qwik config shown on the website. I really like
Stylify but it is a pain reposting the correct css file data every few
minutes. Any help is wanted so I don’t have to go back to tailwind.

I agree it didn’t feel random. It seemed to happen every time I changed the
returned html in the tsx, more specifically the classes on an element.

**Comment by Machy8** [comment]
@petersrule Hi, I am not sure if you still use Stylify but in case you do:

Drop the Unplugin. 
Add bundler into vite config https://stylifycss.com/en/docs/get-started/bundling-files. 

In case you want to split those tasks, run paralel tasks with concurrently like here https://stylifycss.com/en/docs/integrations/remix/

I have been thinking about plugins and integrations and I don't think, it's the right way. So I am dropping them all in favor of bundler.

The longer why: https://github.com/stylify/packages/discussions/274
